tools/breaking-changes/compare.ts

```diff
@@ -16,6 +15,7 @@ const newApiFile = process.argv[3];
 
 const newApiData = JSON.parse(fs.readFileSync(newApiFile, 'utf-8'));
 const oldApiData = JSON.parse(fs.readFileSync(oldApiFile, 'utf-8'));
+const renamedApiLookupData = common.readRenamedApiLookupFile();
```

Comment:
Just to confirm this is not a valid use case.

1. someone forgets to include the breaking change in the renamed-api.json as part of there DoD in the PR
2. we already generated the migrations, so like we ran yarn compare to generate the breaking-change.json
3. after some time the person who forgot to include the BR includes it, and then includes it in the renamed-api.json

even if we `yarn gen-doc`, it won't work due to the breaking-change.json not having the component/class marked as one that is moved, but still shown as 'deleted'.

So the question is, is it fine for us to `yarn compare` again manually to get the truthy breaking-change.json file.
I was thinking if we do `yarn gen-doc`, it should take into account 'new' added changes instead of having to manually run the `yarn compare` command before running the schematics migration package script again.

What do you think of running `yarn compare` at every schematics/docs package script?

----
Reason why I'm saying this is because if we add changes to deleted-api.json, deleted-renamed-api-members.json, then those are taken into account from the get-go. If those are taken into account right away, then the ones that were marked as deleted, but are supposed to be moved, should have the same behavior. What do you think?

Reply:
It's important for the renamed lookup to be performed by the compare script because it is a pre-requisite for comparison.  When an api element is renamed, it's only via the rename lookup info that the compare script can compare the previous and current versions of the renamed api element.   Therefore we can't delay taking into account the rename lookup to after the compare.  If I didn't quite understand your point let's discuss.